docs/content/preview/yugabyte-platform/install-yugabyte-platform/prepare-environment/kubernetes.md

```diff
@@ -156,258 +156,155 @@ Before proceeding, ensure that you have the following:
-   ```
-
-   ```sh
-   docker tag a04fef023c7c gcr.io/dataengineeringdemos/yugabyte/yugaware:2.5.1.0-b153
-   docker tag 721453480a0f gcr.io/dataengineeringdemos/yugabyte/thirdparty-deps:latest
-   docker tag 5a9061639d0a gcr.io/dataengineeringdemos/yugabyte/nginx:1.17.4
-   docker tag 5f1485c70c9a gcr.io/dataengineeringdemos/yugabyte/postgres:11.5
-   docker tag cc866859f8df gcr.io/dataengineeringdemos/prom/prometheus:v2.2.1
-   docker tag caef6233eac4 gcr.io/dataengineeringdemos/janeczku/go-dnsmasq:release-1.0.7
-   ```
-
-1. Push images to the private container registry, as follows:
-
-   ```sh
-   docker push a04fef023c7c 
-   docker push 721453480a0f 
-   docker push 5a9061639d0a 
-   docker push 5f1485c70c9a
-   docker push cc866859f8df 
-   docker push caef6233eac4 
-   ```
-
-   ![img](/images/yp/docker-image.png)
-
-1. Modify the Helm chart `values.yaml` file. You can map your private internal repository URI to `commonRegistry` and use the folder or `project/image_name` and tags similar to the following:
-
-   ```properties
-   image:
-      commonRegistry: "**gcr.io/dataengineeringdemos**"
-      repository: **""**
-      tag: **2.5.1.0-b153**
-      pullPolicy: IfNotPresent
-      pullSecret: yugabyte-k8s-pull-secret
-      thirdparty-deps:
-      		registry: /yugabyte/thhirdparty-deps
-   			tag: **latest**
-   			name: **yugabyte/thirdparty-deps**
-      postgres:
-   			registry: "yugabyte/postgres"
-   			tag: 11.5
-   			name: **postgres**
-      prometheus:
-   			registry: "prom/prometheus"
-   			tag: **v2.2.1**
-   			name: **prom/prometheus**
-      nginx:
-   			registry: "yugabyte/nginx"
-   			tag: **1.17.4**
-   			name: nginx
-      dnsmasq:
-   			registry: "janeczku/go-dnsmasq/"
-   			tag: **release-1.0.7**
-   			name: **janeczku/go-dnsmasq
-   ```
-
-1. Install Helm chart or specify the container registry in YugabyteDB Anywhere cloud provider, as follows:
-
-   ```sh
-   helm install yugaware **.** -f values.yaml
-   ```
+- Login to [quay.io](https://quay.io/) to access the YugabyteDB private registry using the user name and password provided in the secret YAML file. To find the `auth` field, use `base64 -d` to decode the data inside the `yaml` file twice. In this field, the user name and password are separated by a colon. For example, `yugabyte+<user-name>:ZQ66Z9C1K6AHD5A9VU28B06Q7N0AXZAQSR`.
+
+  ```sh
+  docker login -u "your_yugabyte_username" -p "yugabyte_provided_password" quay.
+  docker search quay.io/yugabyte
+  ```
+
+- Install Helm to fetch the YugabyteDB Anywhere Helm chart on your machine. Follow to see [Helm Installation](https://helm.sh/docs/intro/install/).
+
+- Run the following `helm repo add` command to clone the [YugabyteDB charts repository](https://charts.yugabyte.com/).
+
+  ```sh
+  helm repo add yugabytedb https://charts.yugabyte.com
+  ```
+
+- Run the following `helm repo update` command to fetch the updates from the Chart repo.
+
+  ```sh
+  helm repo update
+  ```
```

Comment:
I think we can just combine these two commands in a single instruction. 
This particular section will focus more on container images, than how to add Helm repo and check if it is working.